BuildingBlocks/src/BuildingBlocks.SDK/UriUtils.cs

```diff
@@ -2,6 +2,5 @@
 
 public class UriUtils
 {
-    public static Uri CreateUriFromParts(params string[] parts)
-        => new Uri(parts.Aggregate((a, b) => $"{a.TrimEnd('/')}/{b.TrimStart('/')}"));
+    public static Uri CreateUriFromParts(params string[] parts) => new(parts.Aggregate((a, b) => $"{a.TrimEnd('/')}/{b.TrimStart('/')}"));
```

Comment:
Should be extracted into a method. Currently it's very hard to read and understand

Reply:
Would it be better like this?
```c#
var path = parts.Aggregate((a, b) => string.Format("{0}/{1}", a.TrimEnd('/'), b.TrimStart('/')));
```

Or with the interpolated string from earlier (bc ReSharper recommends me to use it)

Reply:
Hmm, I'm okay with both.